Formula/libp11.rb

```diff
@@ -29,20 +29,20 @@ class Libp11 < Formula
 
   depends_on "pkg-config" => :build
   depends_on "libtool"
-  depends_on "openssl@1.1"
+  depends_on "openssl@3"
 
   def install
     system "./bootstrap" if build.head?
-    system "./configure", "--disable-dependency-tracking",
-                          "--prefix=#{prefix}",
+    system "./configure", *std_configure_args,
+                          "--disable-silent-rules",
                           "--with-enginesdir=#{lib}/engines-1.1"
```

Comment:
Shouldn't this be `--with-enginesdir=#{lib}/engines-3`?

Reply:
Yes, thanks. #125013